devel/gh/Portfile

```diff
@@ -1,54 +1,64 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
-PortGroup           legacysupport 1.1
+PortGroup           golang 1.0
 
-github.setup        cli cli 1.11.0 v
+go.setup            github.com/cli/cli 1.12.0 v
 revision            0
 name                gh
+
+homepage            https://cli.github.com
+
+description         GitHub’s official command line tool
+
+long_description    ${name} is GitHub on the command line. It brings pull \
+                    requests, issues, and other GitHub concepts to the \
+                    terminal next to where you are already working with git \
+                    and your code.
+
 categories          devel
-platforms           darwin
-supported_archs     x86_64
 license             MIT
 maintainers         {l2dy @l2dy} \
                     {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
-
-description         The GitHub CLI
-long_description    gh is GitHub on the command line, and it's now available in beta.
-
-homepage            https://cli.github.com
+installs_libs       no
 
 # oauthClientID in the repository is meant for development
```

Comment:
Please remove this comment if we build gh from source.

Reply:
Removed.